perception/object_recognition/detection/object_merger/src/object_association_merger/data_association/data_association.cpp

```diff
@@ -26,246 +26,236 @@ DataAssociation::DataAssociation() : score_threshold_(0.1)
+    autoware_auto_perception_msgs::msg::ObjectClassification::TRUCK,
+    autoware_auto_perception_msgs::msg::ObjectClassification::UNKNOWN) = 0;
   can_assign_matrix_(
-    autoware_perception_msgs::msg::Semantic::TRUCK, autoware_perception_msgs::msg::Semantic::CAR) =
-    1;
+    autoware_auto_perception_msgs::msg::ObjectClassification::TRUCK,
+    autoware_auto_perception_msgs::msg::ObjectClassification::CAR) = 1;
   can_assign_matrix_(
-    autoware_perception_msgs::msg::Semantic::TRUCK, autoware_perception_msgs::msg::Semantic::BUS) =
-    1;
+    autoware_auto_perception_msgs::msg::ObjectClassification::TRUCK,
+    autoware_auto_perception_msgs::msg::ObjectClassification::BUS) = 1;
   can_assign_matrix_(
-    autoware_perception_msgs::msg::Semantic::BUS,
-    autoware_perception_msgs::msg::Semantic::UNKNOWN) = 0;
+    autoware_auto_perception_msgs::msg::ObjectClassification::BUS,
+    autoware_auto_perception_msgs::msg::ObjectClassification::UNKNOWN) = 0;
   can_assign_matrix_(
-    autoware_perception_msgs::msg::Semantic::BUS, autoware_perception_msgs::msg::Semantic::CAR) = 1;
+    autoware_auto_perception_msgs::msg::ObjectClassification::BUS,
+    autoware_auto_perception_msgs::msg::ObjectClassification::CAR) = 1;
   can_assign_matrix_(
-    autoware_perception_msgs::msg::Semantic::BUS, autoware_perception_msgs::msg::Semantic::TRUCK) =
-    1;
+    autoware_auto_perception_msgs::msg::ObjectClassification::BUS,
+    autoware_auto_perception_msgs::msg::ObjectClassification::TRUCK) = 1;
   can_assign_matrix_(
-    autoware_perception_msgs::msg::Semantic::BICYCLE,
-    autoware_perception_msgs::msg::Semantic::UNKNOWN) = 0;
+    autoware_auto_perception_msgs::msg::ObjectClassification::BICYCLE,
+    autoware_auto_perception_msgs::msg::ObjectClassification::UNKNOWN) = 0;
   can_assign_matrix_(
-    autoware_perception_msgs::msg::Semantic::BICYCLE,
-    autoware_perception_msgs::msg::Semantic::MOTORBIKE) = 1;
+    autoware_auto_perception_msgs::msg::ObjectClassification::BICYCLE,
+    autoware_auto_perception_msgs::msg::ObjectClassification::MOTORCYCLE) = 1;
   can_assign_matrix_(
-    autoware_perception_msgs::msg::Semantic::MOTORBIKE,
-    autoware_perception_msgs::msg::Semantic::UNKNOWN) = 0;
+    autoware_auto_perception_msgs::msg::ObjectClassification::MOTORCYCLE,
+    autoware_auto_perception_msgs::msg::ObjectClassification::UNKNOWN) = 0;
   can_assign_matrix_(
-    autoware_perception_msgs::msg::Semantic::MOTORBIKE,
-    autoware_perception_msgs::msg::Semantic::BICYCLE) = 1;
+    autoware_auto_perception_msgs::msg::ObjectClassification::MOTORCYCLE,
+    autoware_auto_perception_msgs::msg::ObjectClassification::BICYCLE) = 1;
   can_assign_matrix_(
-    autoware_perception_msgs::msg::Semantic::PEDESTRIAN,
-    autoware_perception_msgs::msg::Semantic::UNKNOWN) = 0;
-  can_assign_matrix_(
-    autoware_perception_msgs::msg::Semantic::ANIMAL,
-    autoware_perception_msgs::msg::Semantic::UNKNOWN) = 0;
```

Comment:
@yukkysaito @yukke42 please check the following modification.

- remove ANIMAL label from following 4 matrixes:
  - can_assing_matrix
  - max_dist_matrix
  - max_area_matrix
  - min_area_matrix

Reply:
No problem